projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -205,6 +206,8 @@ if (environment.epdVisualization) {
     ProductConfiguratorTextfieldFeatureModule,
     ProductConfiguratorRulebasedFeatureModule,
 
+    AccountSummaryFeatureModule,
```

Comment:
AccountSummary is a b2b feature, right? If yes, this one should be put inside `if (environment.b2b) {...}`

Reply:
correct